solutions/CSES Graph Girth.mdx

```diff
@@ -26,234 +26,68 @@ This algorithm runs in $\mathcal{O}(N + M)$ time. Since $N$ and $M$ are so small
 The final complexity of this solution is thus $\mathcal{O}(N(N + M))$.
 
 ```cpp
-#include <bits/stdc++.h>
-using namespace std;
- 
-using ll = long long;
-using ld = long double;
-using db = double; 
-using str = string; // yay python!
-
-using pi = pair<int,int>;
-using pl = pair<ll,ll>;
-using pd = pair<db,db>;
-
-using vi = vector<int>;
-using vb = vector<bool>;
-using vl = vector<ll>;
-using vd = vector<db>; 
-using vs = vector<str>;
-using vpi = vector<pi>;
-using vpl = vector<pl>; 
-using vpd = vector<pd>;
-
-#define tcT template<class T
-#define tcTU tcT, class U
-// ^ lol this makes everything look weird but I'll try it
-tcT> using V = vector<T>; 
-tcT, size_t SZ> using AR = array<T,SZ>; 
-tcT> using PR = pair<T,T>;
-
-// pairs
-#define mp make_pair
-#define f first
-#define s second
-
-// vectors
-// oops size(x), rbegin(x), rend(x) need C++17
-#define sz(x) int((x).size())
-#define bg(x) begin(x)
-#define all(x) bg(x), end(x)
-#define rall(x) x.rbegin(), x.rend() 
-#define sor(x) sort(all(x)) 
-#define rsz resize
-#define ins insert 
-#define ft front()
-#define bk back()
-#define pb push_back
-#define eb emplace_back 
-#define pf push_front
-
-#define lb lower_bound
-#define ub upper_bound 
-tcT> int lwb(V<T>& a, const T& b) { return int(lb(all(a),b)-bg(a)); }
-
-// loops
-#define FOR(i,a,b) for (int i = (a); i < (b); ++i)
-#define F0R(i,a) FOR(i,0,a)
-#define ROF(i,a,b) for (int i = (b)-1; i >= (a); --i)
-#define R0F(i,a) ROF(i,0,a)
-#define trav(a,x) for (auto& a: x)
+#include <iostream>
```

Comment:
just change everything to `#include <bits/stdc++.h>` - tbh idt we should detemplify that...

Reply:
I think this is fine... up to you whether to change it or not

Reply:
> naaaaaaaaahhhh it's fine right? -- as long as there's not too many unnecessary headers

(see Frod's comment above. Unless enough people prefer bits/stdc++.h, what is currently here is probably fine)